README.md

```diff
@@ -51,16 +51,36 @@ end
 
 Generally speaking, you as a user of AMS will write (or generate) these
 serializer classes. If you want to use a different adapter, such as a JsonApi, you can
-change this in an initializer:
+change this in your render, or in an initializer:
 
 ```ruby
+#class
 ActiveModel::Serializer.config.adapter = ActiveModel::Serializer::Adapter::JsonApi
-```
 
+#symbol
+ActiveModel::Serializer.config.adapter = :json_api
+```
 or
 
 ```ruby
-ActiveModel::Serializer.config.adapter = :json_api
+render json: @posts, adapter: :json_api
+```
```

Comment:
Despite of being simple, would be nice to have in our new docs, maybe some same article on "How to use multiple adapters on the same app" or "How to use a specific adapter on a request"